test/solveEquation/solveEquation.test.js

```diff
@@ -14,8 +14,9 @@ function testSolve(equationString, outputStr, debug=false) {
   else {
     lastStep = steps[steps.length -1].newEquation.print();
   }
-  it(equationString + ' -> ' + outputStr, () => {
+  it(equationString + ' -> ' + outputStr, (done) => {
     assert.equal(lastStep, outputStr);
+    done();
```

Comment:
I'm not that familiar with `done()`

how does it fix the long tests? I googled around a bit and it seems like they're mostly used for needing to wait for asychnonous things to finish